algorithms/french/stemmer.tt

```diff
@@ -426,8 +432,6 @@ Step 4: Residual suffix
         <DD>replace with <B><I>i</I></B>
     <DT><B><I>e</I></B>
         <DD>delete
-    <DT><B><I>ë</I></B>
-        <DD>if preceded by <B><I>gu</I></B>, delete
```

Comment:
Was this an accidental deletion?  This rule is still present.

Reply:
It was intentional, because the rule was removed: https://github.com/snowballstem/snowball/commit/86ceab9bc5b7a8e6ed911055e155572290489645#diff-692b850ec0e476ee9906b7d54863297cL206.

Reply:
Oh sorry, I was looking at a checkout of the snowball repo on a machine I hadn't pulled the latest changes on.

This is gone because this case now gets handled by being converted to `e` then removed.